cro-clib/src/jsonrpc.rs

```diff
@@ -264,7 +264,7 @@ unsafe fn create_rpc(
     network_id: u8,
     progress_callback: Option<ProgressCallback>,
     user_data: *const std::ffi::c_void,
-    mock: bool,
+    _mock: bool,
```

Comment:
remove this parameter, I guess mock mode is controlled by compile feature. I guess you need to add feature "mock-enclave" to cro-clib too.

Reply:
yeah, then the interface aguments will be changed and other applications using the cro-clib will also need to be fixed, it's another issue i think, so I keep this.